res/css/views/rooms/_EventTile.pcss

```diff
@@ -648,6 +648,32 @@ $left-gutter: 64px;
             margin-top: 0;
             margin-bottom: 0;
         }
+
+        /* EventTile on every layout */
+        .mx_EventTile & {
+            /* Paragraph below ul or ol */
+            :is(ul, ol) + p {
+                margin-block: $spacing-16;
+            }
+        }
+
+        /* EventTile on compact modern layout only */
+        .mx_MatrixChat_useCompactLayout .mx_EventTile[data-layout="group"] & {
```

Comment:
I thought this would make it easier to understand how differently the values of margin is set. Since I don't really care where this block should be placed, going to revert.

Reply:
https://github.com/matrix-org/matrix-react-sdk/pull/10305/commits/98adc1154ab92f830045d2bf80965f0d955df3c0